index.js

```diff
@@ -109,7 +119,8 @@ require('util').inherits(RingPop, EventEmitter);
 
 RingPop.prototype.destroy = function destroy() {
     this.destroyed = true;
-    clearTimeout(this.gossipTimer);
+    this.gossip.stop();
```

Comment:
nitpick: favor having a method called `.destroy()`

Every single thing that has a side effect in terms of keeping the event loop open in node core has an `unref()` or `destroy()` method.

With the exception that servers have a `close()` method.